### DISCUSSION TITLE
How to use it with pyinstaller?

### DISCUSSION THREAD

**Discussion by ghost** [openvinotoolkit/openvino#16104] [Q&A] [answered]
I want to distribute some openvino models in a standalone application for windows, mac and linux?
It does not seem to be compatale with pyinstaller. 

What is the best path fwd?

**Comment by ilya-lavrenov** [comment, marked as answer]
> It does not seem to be compatale with pyinstaller.

Why not? pyinstaller seems to be universal approach working on all OSes
